contracts/src/subnet/SubnetActorCheckpointingFacet.sol

```diff
@@ -32,28 +33,20 @@ contract SubnetActorCheckpointingFacet is SubnetActorModifiers, ReentrancyGuard,
 
         bytes32 checkpointHash = keccak256(abi.encode(checkpoint));
 
-        if (checkpoint.blockHeight <= s.lastBottomUpCheckpointHeight) {
-            revert BottomUpCheckpointAlreadySubmitted();
-        }
-
-        // When a bottom up msg batch becomes full, a bottom up checkpoint will be
-        // created before the next checkpoint period is reached.
-        if (checkpoint.blockHeight <= s.lastBottomUpCheckpointHeight + s.bottomUpCheckPeriod) {
-            // validate signatures and quorum threshold, revert if validation fails
-            validateActiveQuorumSignatures({signatories: signatories, hash: checkpointHash, signatures: signatures});
+        // validate signatures and quorum threshold, revert if validation fails
+        validateActiveQuorumSignatures({signatories: signatories, hash: checkpointHash, signatures: signatures});
```

Comment:
For the future, https://github.com/consensus-shipyard/ipc/pull/791 and this PR are closely interrelated. The former is adding checks that this PR is now deleting. It would've been simpler to push these changes to the previous PR, to prevent reviewers from reviewing stale logic.